src/resources/postcss/tribe-common-admin/settings/_form.pcss

```diff
@@ -23,6 +23,7 @@
 	}
 
 	#tec-settings-form,
+	#tec-tickets-settings-form,
```

Comment:
Why do we need a new class here? Can we just use `#tec-settings-form` since it is not plugin-specific?